packages/rocketchat-importer-slack-users/server/importer.js

```diff
@@ -0,0 +1,129 @@
+import {
+	Base,
+	ProgressStep,
+	Selection,
+	SelectionUser
+} from 'meteor/rocketchat:importer';
+
+export class SlackUsersImporter extends Base {
+	constructor(info) {
+		super(info);
+
+		this.csvParser = Npm.require('csv-parse/lib/sync');
+		this.userMap = new Map();
+		this.admins = []; //Array of usernames of the admins
+	}
+
+	prepare(dataURI, sentContentType, fileName) {
+		super.prepare(dataURI, sentContentType, fileName, true);
+
+		super.updateProgress(ProgressStep.PREPARING_USERS);
+		const uriResult = RocketChatFile.dataURIParse(dataURI);
+		const buf = new Buffer(uriResult.image, 'base64');
+		const parsed = this.csvParser(buf.toString());
+
+		parsed.forEach((user, index) => {
+			// Ignore the first column
+			if (index === 0) {
+				return;
+			}
+
+			const id = Random.id();
+			const username = user[0];
+			const email = user[1];
+			let isBot = false;
+			let isDeleted = false;
+
+			switch (user[2]) {
+				case 'Admin':
+					this.admins.push(username);
+					break;
+				case 'Bot':
+					isBot = true;
+					break;
+				case 'Deactivated':
```

Comment:
Any chance it'll ever be more than one of these?

Reply:
No clue, these are the only ones that I have seen in the export that contained 5000+ users..

Reply:
What I mean by this is can it ever be bot and deactivated?  or admin and deactivated?  or admin, bot and deactivated?

Reply:
Ah okay. No, it is a single value field.